web/lib/src/helpers/extensions.dart

```diff
@@ -103,3 +104,14 @@ extension UriToURL on Uri {
     }
   }
 }
+
+extension NodeExtension on Node {
+  /// Returns [childNodes] as a modifiable [List]
+  List<Node> get childNodesAsList => JSLiveNodeListWrapper(this, childNodes);
```

Comment:
I don't believe this is a mutable list. Should this be `JSImmutableListWrapper`?